### DISCUSSION TITLE
Nested SBOMs

### DISCUSSION THREAD

**Discussion by ctron** [guacsec/trustify#197] [Ideas]
It's possible to reference from one SBOM to another. I don't think we support that right now. But we should.

Re-using content that was already authored. This might also make it easier to handle things like products composed/re-using other products.

**Comment by bobmcwhirter** [comment]
I do think this pushes towards your idea of a unified Node-relatesto-Node table, for a variety of nodes.

sbom-contains-package. sbom-contains-sbom, etc
